rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml

```diff
@@ -427,7 +431,7 @@ delete over _bulk:
   - match: {items.0.delete.result: deleted}
   - match: {items.1.delete.result: deleted}
   - match: {items.2.delete.status: 404}
-  - match: {items.2.delete.error.reason: "invalid id [not found ++ not found] for index [id_generation_test] in time series mode"}
+  - match: {items.2.delete.error.reason: "invalid id [not found ++ not found] for index [id_generation_test] in time_series mode"}
```

Comment:
I thought that older versions still run the test with their local (old) version of the error message. The cluster feature should only affect mixed cluster tests. If so, I'd argue that the test coverage loss is marginal, since the tests will fail in both the old and the new version - with slightly different messages.

Is my understanding correct?

Reply:
We will lose test coverage of these tests with updated error message against all versions prior to 8.18.0 and I think that we should avoid that? I think we should avoid that a change in error message causes as to not run these test against older versions. This is why I think we should update the assertion to not case whether `time series` or `time_series` is included in the error.